ntc_templates/templates/alcatel_aos_show_interfaces_port.textfsm

```diff
@@ -0,0 +1,16 @@
+Value PORT ((\d+\/?)+)
+Value WTR_RUNNING (\#?)
+Value PERMANENT_SHUTDOWN (\*?)
+Value ADMIN_STATUS (\S+)
+Value LINK_STATUS (\S+)
+Value VIOLATIONS (\S+)
+Value RECOVERY_TIME (\d+)
+Value RECOVERY_MAX (\d+)
+Value WTR (\d+)
+Value ALIAS (\S+)
+
+Start
+  ^\s*[-+]+\s*$$
+  ^\s*${WTR_RUNNING}${PERMANENT_SHUTDOWN}${PORT}\s+${ADMIN_STATUS}\s+${LINK_STATUS}\s+${VIOLATIONS}\s+${RECOVERY_TIME}\s+${RECOVERY_MAX}\s+${WTR}\s+${ALIAS}\s*$$ -> Record
```

Comment:
```suggestion
Value WTR_TIME (\d+)
Value ALIAS (\S+)

Start
  ^\s*-+\+
  ^\s*${WTR_RUNNING}${PERMANENT_SHUTDOWN}${PORT}\s+${ADMIN_STATUS}\s+${LINK_STATUS}\s+${VIOLATIONS}\s+${RECOVERY_TIME}\s+${RECOVERY_MAX}\s+${WTR_TIME}\s+${ALIAS}\s*$$ -> Record
```

Use the separator pattern that @jmcgill298 supports.

Improve self-description of capture group by making `WTR` >> `WTR_TIME`.

Thank you!